calico_node/filesystem/etc/calico/confd/config/custom_filters.cfg

```diff
@@ -0,0 +1,5 @@
+# Generated by confd
```

Comment:
There aren't any other files in calico_node/filesystem/etc/calico/confd/config/, so I would guess that this file should not be checked into Git.  (Unless there is a reason why different handling is needed for this file, than all of the other confd-generated files?  In which case please explain that.)